src/main/java/org/broadinstitute/hellbender/utils/read/GoogleGenomicsReadToGATKReadAdapter.java

```diff
@@ -299,6 +299,16 @@ public void setMappingQuality( final int mappingQuality ) {
     }
 
     @Override
+    public int getLength() {
```

Comment:
Remove the default implementation of `getLength()` in `GATKRead`? (since it will always be inefficient given that the contract for `getBases()` states that it will always make a defensive copy).